bin/deployment/bash_functions

```diff
@@ -158,13 +173,17 @@ init_mysql_jdbc() {
     local db_host="${DBHOSTNAME:-localhost}"
     local password="$DBPASSWORD"
     [ -z "$password" ] && password=''
-    JDBC_DRIVER="com.mysql.jdbc.Driver"
-    JDBC_JAR="/usr/share/java/mysql-connector-java.jar"
+
+    select_jdbc_jar "/usr/lib/java/mariadb-java-client.jar" "/usr/share/java/mysql-connector-java.jar"
+    evalrc $? "No JDBC jar file available for MariaDB/MySQL"
+
+    JDBC_DRIVER="com.mysql.cj.jdbc.Driver"
     JDBC_URL="jdbc:mysql://$db_host/$db_name"
 
-    # Change once the mariadb-java-client RPM is available in EPEL
+    # FIXME:
+    # Use the mariadb driver and URL when the mariadb-java-client RPM is available in EL7 and the
+    # Liquibase errors are fully resolved with the newer drivers
```

Comment:
Did you hit these liquibase errors after all?

Reply:
Specifically when changing the JDBC_DRIVER to MariaDB's. It doesn't like something with a few places we use NULL. I forget the details and I didn't look too deeply into it.